.github/workflows/benchmark.yml

```
@@ -23,21 +23,23 @@ jobs:
       - name: Setup Miniforge
         uses: conda-incubator/setup-miniconda@v3
         with:
-          miniforge-version: "24.1.2-0"
           activate-environment: asv-bench
+          environment-file: environment.yml
       - name: Install Requirements
         run: pip install -r requirements.txt
```

```suggestion
        run: python -m pip install --requirement requirements.txt
```
Using `pip` as module is a best practice, and also long form for attributes.